.github/workflows/spelling.yaml

```
@@ -0,0 +1,29 @@
+name: Check Spelling
```

```suggestion
name: Spelling
```

These are shown in the GitHub "Checks" UI, so I think prefixing the name with "Check" is redundant and/or confusing.  I'd also like the name to start with "Spell" since it's the more meaningful term.  How about just "Spelling"?  I would also be OK with some form of "Spell Check" or "Spelling Checker", since I know "Spell Check" is a common phrase.

---

Once this PR is merged, and the previous check is disabled, we will also need to update the required check name in the repo settings.